docs/developer-guide/nodejs-api.md

```diff
@@ -551,6 +582,22 @@ The information available for each linting message is:
 * `fix` - an object describing the fix for the problem (this property is omitted if no fix is available).
 * `suggestions` - an array of objects describing possible lint fixes for editors to programmatically enable (see details in the [Working with Rules docs](./working-with-rules.md#providing-suggestions)).
 
+You can get the suppressed messages from the pervious run by `getSuppressedMessages()` method. If there is not a previous run, `getSuppressedMessage()` will return an empty list.
+
+```js
+const Linter = require("eslint").Linter;
+const linter = new Linter();
+
+const messages = linter.verify("var foo = bar; // eslint-disable-line -- Need to suppress", {
+    rules: {
+        semi: 2
```

Comment:
```suggestion
        semi: ["error", "never"]
```

To produce an error in the provided source code (default for `semi` is `"always"`, so there would be no suppressed messages as the provided source code would be valid).